src/home_work_14_1.ts

```diff
@@ -0,0 +1,96 @@
+interface Note {
+  id: string;
+  title: string;
+  content: string;
+  createdAt: Date;
+  updatedAt: Date;
+  status: "completed" | "pending";
+  confirmEdit: boolean;
+}
+
+class TodoList {
+  notes: Note[];
+
+  constructor() {
+    this.notes = [];
+  }
+
+  addNote(note: Note) {
```

Comment:
Always remember to explicitly add type annotations for return function values. It's a great practice!